docs/developer-guide/release.md

```diff
@@ -48,22 +48,29 @@ _Note: While the release is running, don't shut your laptop or switch between re
 
 ## Prepare the release
 1. Do a docker pull: `docker pull obolibrary/odkfull:dev`
-1. Pull master
-1. Run command: `cd mondo/src/ontology/` (navigate to folder on your computer)
-1. `sh run.sh make IMP=false all -B` 
+    - note, this step should not be required anymore.    
+1. Pull master, and navigate to the `mondo/src/ontology` folder on your computer
+1. **Run command: `sh run.sh make IMP=false all -B`** 
     - note, this takes 1+ hour(s)
     - note that we are using the dev image as it is always up to date with the Python dependencies.
-1. Make sure you see ‘release finished’ after the command has run
-1. Open mondo.owl and mondo.obo and check the latest changes are there and it looks reasonable
-1. Run `sh run.sh make prepare_release_direct`
-1. WARNING: TO BE UPDATED Review the file `src/ontology/reports/mondo_release_diff.md`. 
-    - There is now a new QC section up top, `---START LOG:` to `---END LOG:`. Review the text and _delete it from the file_ if there is no suspicious output.
-    - note that this file is 'gitignore', ie one only sees it if they are the one running the release.
+1. Review before the next step:
+    1. Make sure you see ‘release finished’ after the command has run
+    1. Open mondo.owl and mondo.obo and check the latest changes are there and it looks reasonable
+1. **Run `sh run.sh make prepare_release_direct`**
+    1. reports files will be created after running this command.
 1. Commit changes to a branch
-      1. Create a branch and commit the changes on the branch
-      1. Do a pull request (PR)
-      1. Wait for GitHub Actions/QC to pass
-      1. Merge PR
+    1. Create a branch and commit the changes on the branch
```

Comment:
Personally, I like to create a new branch before running any commands since I find it easier and safer to delete a branch, than run the risk of accidentally committing any files to `master`.

Reply:
updated